src/index.ts

```diff
@@ -1,4 +1,4 @@
-import * as contentstack from './lib/contentstack';
+import contentstack from './lib/contentstack';
 export * from './lib/types';
 
-export default contentstack;
+export { contentstack as stack };
```

Comment:
@vkalta Exporting contentstack as stack will cause confusion in method chaining calls the initialization will be `stack.stack( ... )`
Can you please confirm this?

Reply:
I have tested it. It works fine @nadeem-cs

Reply:
The whole point is to not do stack.stack and just stack @nadeem-cs